boards/x86/intel_ehl/Kconfig.defconfig

```diff
@@ -20,7 +20,7 @@ config SHELL_BACKEND_SERIAL_INTERRUPT_DRIVEN
 endif
 
 config HEAP_MEM_POOL_SIZE
-	default 32768 if ACPI
+	default 2097152 if ACPI
```

Comment:
Maybe add also to RPL?

Reply:
I see that this `HEAP_MEM_POOL_SIZE` for RPL is in _defconfig, probably it makes sense to unify all Intel boards.

Reply:
> Maybe add also to RPL?

RPL does this in intel_rpl_s_crb_defconfig. We should probably make this consistent across all platforms, but I didn't change it in this PR.

Reply:
> I see that this `HEAP_MEM_POOL_SIZE` for RPL is in _defconfig, probably it makes sense to unify all Intel boards.

Agree. I think `Kconfig.defconfig` is the right place.